packages/website/src/components/Playground.module.css

```diff
@@ -50,10 +50,15 @@
   z-index: 1;
 }
 
-.tabCode {
+.tabCode,
+.playgroundInfoContainer {
   height: calc(100% - 41px);
 }
 
+.playgroundInfoContainer {
```

Comment:
[Question] Looks like `.playgroundInfoContainer` into a more generalized version of `.tabCode`. Could we merge them together and replace `.tabCode` with `.playgroundInfoContainer` altogether? I tried in the deploy preview (adding the `overflow: auto` to `.tabCode`'s styles) and it seemed to not harm anything.